wiki/Client/Interface/en.md

```diff
@@ -1,78 +1,76 @@
+- \[10\] The jukebox. In the main menu, it plays songs in a random order. See below for more information about the buttons in this menu.
+- \[11\] The number of [beatmap difficulties](/wiki/Beatmap/Difficulty) available (despite what it says), how long you have been playing for, and your system clock.
+- \[12\] Your profile. Clicking on it shows the [User Options](#user-options).
+
+---
+
+The jukebox has the following buttons:
+
+| Button | Description |
+| :-: | :-- |
+| ![](img/previous-track.jpg "Previous Track") | Previous track |
+| ![](img/play.jpg "Play") | Play |
+| ![](img/pause.jpg "Pause") | Pause |
+| ![](img/stop.jpg "Stop") | Stop and reset the current song to its beginning |
+| ![](img/next-track.jpg "Next Track") | Next track |
+| ![](img/view-song-info.jpg "View Song Info") | Toggle between having the information for the current song showing permanently, or fading out after a little while. |
+| ![](img/jump-to-window.jpg "Jump To Window") | Jump to a specific song. You can search for songs or filter by collection. |
+
+The jukebox can also be controlled using [keyboard shortcuts](/wiki/Client/Keyboard_shortcuts#jukebox).
 
 ## User Options
 
 ![](img/user-options.jpg "User Options")
 
-Access this screen by clicking your profile at the top left of the main menu. You cannot access the Chat Consoles while viewing the user option screen. You can select any item by pressing the corresponding number on the option:
+Access this screen by clicking on your profile at the top left of the main menu. You can select any item by pressing its corresponding number:
 
-1. `View Profile`: Opens up your profile page in your default web browser.
-2. `Sign Out`: Sign out of your account (after signing out, the [Options](/wiki/Client/Options) sidebar will prompt you to sign in).
-3. `Change Avatar`: Open up the edit avatar page in your default web browser.
-4. `Close`: Close this dialog
+- `1. View Profile`: View your profile page on the website.
+- `2. Sign Out`: Sign out of your account. After signing out, the game will prompt you to sign back in.
+- `3. Change Avatar`: Open the [edit avatar page](https://osu.ppy.sh/home/account/edit#avatar) on the website.
+- `4. Close`: Close this dialog.
 
 ## Play Menu
 
 ![](img/play-menu.jpg "Play Menu")
 
-- Click `Solo` (`P`) to play alone.
-- Click `Multi` (`M`) to play with other people. You will be directed to the [Multi](/wiki/Gameplay/Multiplayer) Lobby (see below).
+After selecting `Play` in the main menu, you can choose between three options:
+
+- Click `Solo` (or press `P`) to play on your own. This leads you to the [Song Selection Screen](#song-selection-screen).
+- Click `Multi` (or press `M`) to play with other people. You will be directed to the [Multi](/wiki/Gameplay/Multiplayer) Lobby.
 - Click `Back` to return to the main menu.
 
-## Multi Lobby
+## Multiplayer Lobby
 
 *Main page: [Multi](/wiki/Gameplay/Multiplayer)*
```

Comment:
In my opinion, it is better to call it `Multiplayer` throughout the whole wiki. But I think this is not in the scope of this PR.

Reply:
yeah that's a small follow-up action for me